### ISSUE TITLE
Improved navigation with TAB in properties window

### ISSUE THREAD

**Issue by SOTFB** [sabrogden/Ditto#561]
When viewing the properties window of a clip (ALT + ENTER) and then navigating with TAB between all the inputs, the TAB order is not what one would expect. It should begin in the first input which is "Hot key" and then go from top left to bottom right:
_Hot key --> Win --> Hotkey available globally --> Never auto delete --> Quick paste text -->_ and so on downward.
As of now it's not a random sequence each full iteration ofc, but appears so at first, since it now "jumps" in an unintuitive way.

Please also decide if it's hotkey or hot key 🙃

Love Ditto! Thanks a lot! 🙏
